src/test/java/marquez/api/DatasetSerializationTest.java

```diff
@@ -11,8 +11,10 @@
 import java.util.TimeZone;
 import java.util.UUID;
 import org.junit.BeforeClass;
+import org.junit.Ignore;
 import org.junit.Test;
 
+@Ignore("Need to figure out why this started to fail")
```

Comment:
Sounds good. Mind opening an issue and adding a comment that references the issue #?

Reply:
Sure!
https://github.com/MarquezProject/marquez/issues/140

Reply:
In the future, we should make sure all tests are passing locally before opening the PR and not rely on CircleCI as the only gate for test errors. If the tests do not reliably pass locally, it will cause issues for other developers

Reply:
I think we're already following that protocol. Tests marked ignore will not run either locally or remotely.

I do like that we're handling flaky/failing tests as we see them and not punting them for later. I think finding the root cause of this could be interesting, especially since it's passing in CircleCI and not locally.